dev/devicelab/manifest.yaml

```diff
@@ -341,6 +341,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/ios"]
 
+  android_stack_size_test:
+    description: >
+      Measures increased stack size for one stateless widget during the rendering pipeline.
+    stage: devicelab
+    required_agent_capabilities: ["linux/android"]
```

Comment:
is there a way to specify which device it will run on? This benchmark may only make sense if it is running on the same device

Reply:
32 vs. 64 bits could change the result, but do we have any 32-bit Android devices in the lab?

Reply:
I believe all the `linux/android` devices are moto g4.

Reply:
You can delete this entry altogether. There are no longer any linux bots running the cocoon devicelab runner (which is what this config file is for), and thus the task will add a column to the build dashboard that stays grey forever.